credentials.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+"""
+credentials contains the CredentialsStore class, which responsible for persisting access tokens to disk.
+"""
+
+import logging
+import os
+from pathlib import Path
+
+DEFAULT_CREDENTIALS_PATH = Path(os.path.expanduser('~')) / '.bwapi' / "credentials.txt"
+
+logger = logging.getLogger("bwapi")
+
+
+class CredentialsStore:
```

Comment:
You are inventing a file format when there are plenty of existing ones available. I would suggest using [ConfigParser](https://docs.python.org/3.7/library/configparser.html) which can be treated like a dictionary and is built in to python.

Reply:
Using something like this would also allow you to provide structured metadata if there is a need for it in future. You could use the sections to support multiple accounts too.

Reply:
I'm not inventing anything, I using the existing format to maintain backward compatibility. You're right that this isn't a good format, and I'd love to change it, but that's not something I want to do in this PR.